src/notification.rs

```diff
@@ -133,7 +148,7 @@ impl fmt::Debug for Notification {
                self.get_event(),
                self.get_button_id(),
                self.get_scene_id(),
-               self.get_notification(),
+               self.get_notification_code(),
```

Comment:
I tried to add that, but nothing is actually maintained.

For example, when adding a node, no state message occurs until I press the button on the device to add, and then I get a flurry of state change messages:
ControllerStateStarting, ControllerStateInProgress, ControllerStateCompleted (and that sequence repeats a few times).